### DISCUSSION TITLE
Add a link to the original RSS feed

### DISCUSSION THREAD

**Discussion by shooftie** [SamTV12345/PodFetch#421] [Ideas]
It would be really helpful to have a link somewhere to the original feed rather than just the "PodFetch" proxied feed. This would help with debugging...

![Screenshot 2023-11-13 at 12 05 25](https://github.com/SamTV12345/PodFetch/assets/2161844/f7894820-13bb-4065-a5aa-9b81289057f2)

**Comment by shooftie** [comment]
Great work!

<img width="939" alt="Screenshot 2023-11-15 at 21 29 41" src="https://github.com/SamTV12345/PodFetch/assets/2161844/222b2fc1-90d9-4b31-aa6a-8029335616e3">

**Comment by SamTV12345** [comment]
Thanks. Your issues are really helpful. Haven't seen such detailed issues in a long time.

**Reply by shooftie** [reply]
Glad to be of help. I will keep them coming, feel free to triage them as you see fit. If you need any more detail/mockups, just let me know and I'll do what I can.
